.github/workflows/lint.yaml

```diff
@@ -16,7 +16,9 @@ jobs:
       uses: actions/checkout@master
 
     - name: Vale
-      uses: errata-ai/vale-action@v1.3.0
+      # We want support for at least vale v2.15.3
+      #uses: errata-ai/vale-action@v1.3.0
+      uses: errata-ai/vale-action@master
```

Comment:
It was pinned because we had a breaking change at some point. Might be worth pinning again, and updating intentionally but I don't feel strongly about that.

(please remove commented out line 20, source control can give us that back if we need it)

Reply:
I've changed the comment to explain better why we're using `master`.
And removed the commented out line - thanks for spotting that, it was a pure mistake.